static/src/editors/RequestEditorModal.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div class="modal" tabindex="-1" role="dialog">
+    <div class="modal-dialog large-modal" role="document">
+      <div class="modal-content">
+        <div class="modal-header border-bottom-0">
+          <i class="fa fa-exchange-alt mr-2 mt-3"></i>
+          <div class="modal-title">
+            <h3 class="modal-title">Obtenir les droits de rédaction du questionnaire</h3>
+          </div>
+          <button type="button"
+                  class="close"
+                  data-dismiss="modal"
+                  aria-label="Close">
+          </button>
+        </div>
+
+        <div class="modal-body">
+          <div class="row row-cards row-deck">
+            <div class="col">
+              <div class="card">
+                <div class="card-body">
+                  <h4 class="mb-0">Contactez votre collègue {{ questionnaire.editor.first_name }} {{ questionnaire.editor.last_name }}</h4>
+                  (<a :href="'mailto:' + questionnaire.editor.email">{{ questionnaire.editor.email }}</a>)
+                  <p class="mt-4">Pour modifier ce questionnaire, votre collègue doit vous transférer les droits.</p>
+                </div>
+                <img :src="'/static/img/call-for-help.png'" alt="appel à l'aide">
```

Comment:
Ici ya pas besoin du ":", on peut ecrire juste : 
`src="/static/img/call-for-help.png"`